gremlin/templates/chao-deployment.yaml

```diff
@@ -56,6 +56,12 @@ spec:
                 secretKeyRef:
                   name: {{ include "gremlin.secretName" . }}
                   key: GREMLIN_TEAM_SECRET
+{{- end }}
+{{- if .Values.gremlin.proxy.url }}
+            - name: https_proxy
+              value: {{ .Values.gremlin.proxy.url }}
+            - name: no_proxy
```

Comment:
don't route Kubernetes API requests through the proxy. If we ever need to customize this behavior (which we haven't so far), there's room under the `gremlin.proxy` object to specify hosts to exclude from proxying.

Reply:
good catch